pom.xml

```diff
@@ -27,19 +27,14 @@
         <version.common-utils>1.1</version.common-utils>
         <version.commons-lang3>3.9</version.commons-lang3>
         <version.commons-net>3.6</version.commons-net>
-        <version.geowave>0.9.5</version.geowave>
+        <version.geowave>1.1.0</version.geowave>
         <version.hadoop>3.1.1</version.hadoop>
-        <version.jts>1.13</version.jts>
+        <version.jts>1.16.0</version.jts>
```

Comment:
Looks like there's a 1.16.1 available. Should we use that instead?

Reply:
I tried to use the one that our version of GeoWave was using to be consistent with them.  I'll update it and if all the tests pass I'll push an update.